modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -614,16 +671,34 @@ return view.extend({
 			so.value(ipv4, ipaddrs[ipv4] ? '%s (%s)'.format(ipv4, ipaddrs[ipv4]) : ipv4);
 		});
 
-		so = ss.option(form.Value, 'leasetime', _('Lease time'));
+		so = ss.option(form.Value, 'leasetime', 
+			_('Lease time'),
+			_('Host-specific lease time, e.g. <code>2m</code>, <code>3h</code>, <code>5d</code>.'));
 		so.rmempty = true;
 
-		so = ss.option(form.Value, 'duid', _('DUID'));
+		so = ss.option(form.Value, 'duid',
+			_('DUID'),
+			_('The DHCPv6-DUID of this host.'));
 		so.datatype = 'and(rangelength(20,36),hexstring)';
 		Object.keys(duids).forEach(function(duid) {
 			so.value(duid, '%s (%s)'.format(duid, duids[duid].hostname || duids[duid].macaddr || duids[duid].ip6addr || '?'));
 		});
 
-		so = ss.option(form.Value, 'hostid', _('IPv6 suffix (hex)'));
+		so = ss.option(form.Value, 'hostid',
+			_('IPv6-Suffix (hex)'),
+			_('The IPv6 interface identifier (address suffix) as hexadecimal number (max. 8 chars).'));
```

Comment:
We could add her the following line to check the datatype?
`o.datatype = 'hex(8)';`

Reply:
Tried this. When should the error checking occur? Nothing happened in the GUI with this. Only see this in one other place in the code (and I don't use mwan3).  Browser console stackdumps with `SyntaxError: Unhandled token "hex"`. Whatever you were inspired by is broken. Doc refers to non-existent: `LuCI.validation`, which should be linked, but isn't(?).

Reply:
Although the above might not fully relate to how dnsmasq works in openwrt...

So if we really need to limit to a static 16 bit IPv6 suffix (suffixes can be longer...) then e.g.

`		so.datatype = 'and(rangelength(0,8),hexstring)';`